package/MDAnalysis/core/universe.py

```diff
@@ -243,114 +357,68 @@ class Universe(object):
     bonds, angles, dihedrals
         master ConnectivityGroups for each connectivity type
 
+    .. versionchanged:: 0.20.0
+        Universe() now raises an error. Use Universe(None) or :func:`Universe.empty()` instead.
     """
 
-    def __init__(self, *args, **kwargs):
-        # Store the segments for the deprecated instant selector feature.
-        # This attribute has to be defined early to avoid recursion in
-        # __getattr__.
-        self._instant_selectors = {}
-        # hold on to copy of kwargs; used by external libraries that
-        # reinitialize universes
-        self._kwargs = copy.deepcopy(kwargs)
-
-        # managed attribute holding Reader
-        self._trajectory = None
+    
+    def __init__(self, topology, *coordinates, all_coordinates=False,
```

Comment:
So Python 2.7 doesn't support this syntax, which is why we have the fugly `*args, **kwargs` syntax.  So if we want this merged before the Py3 switch, we can change this to args/kwargs (and lose readability for a while) else we can put this PR on ice for a while and merge it once we've dropped Py2.

Reply:
The only visible changes are to the more articulate argument documentation, so it's probably more efficient to just wait until Python 2 finally goes away. Same with #2333